src/module/module-collection.js

```diff
@@ -53,6 +53,14 @@ export default class ModuleCollection {
 
     parent.removeChild(key)
   }
+
+  isRegistered (path) {
+    const parent = this.get(path.slice(0, -1))
```

Comment:
Trust me, I am not trying to create modules with mkdir :smiley:
I just wanted to say that the responsibility lies with the caller to ensure the correctness of the arguments.